apps/test/unit-tests.js

```diff
@@ -28,6 +28,7 @@ describe('unit tests', function () {
   clearTimeoutsBetweenTests();
   // Uncomment and import this line to perform additional test cleanup validation.
   // In the long term, this should be permanently enabled or removed.
-  // enforceDocumentBodyCleanup({checkEveryTest: true});
+  // enforceDocumentBodyCleanup({checkEveryTest: true}, () => {
   runnable.forEach(testsContext);
+  // });
```

Comment:
Could be worth checking that the right thing still happens when the above lines are uncommented, since enforceDocumentBodyCleanup might now appear twice in a single stack trace. a few comments inline within `enforceDocumentBodyCleanup` on this topic.

Reply:
You are correct that `enforceDocumentBodyCleanup` does not support nested calls.  I'm going ahead and removing these comments for a top-level call for now, since enabling it is probably still a ways off and the utility is more useful when it's actually put into practice on a per-file basis (in fact, that might be a better way to move toward enabling it everywhere).